pkg/port/iscsit/iscsid.go

```diff
@@ -466,6 +471,7 @@ func (s *ISCSITargetDriver) iscsiExecLogin(conn *iscsiConnection) error {
 }
 
 func iscsiExecLogout(conn *iscsiConnection) error {
+	log.Infof("Logout request received from initiator: %q, address: %v", conn.session.Initiator, conn.conn.RemoteAddr().String())
```

Comment:
session can be null here.. we can't do this

Reply:
ok